### ISSUE TITLE
Unexpected 500 http code on previously working character endpoint

### ISSUE THREAD

**Issue by cvweiss** [esi/esi-issues#1060]
# Bug

https://esi.evetech.net/v4/characters/2029709492/

The above URL has worked in the past, as I've fetched the name, security status, etc and have stored this information in my database.

## Request

    GET https://esi.evetech.net/v4/characters/2029709492/

### Status Code

`500`

### Headers

```
< HTTP/2 500 
< date: Sat, 03 Nov 2018 12:53:01 GMT
< content-type: application/json; charset=UTF-8
< content-length: 49
< access-control-allow-credentials: true
< access-control-allow-headers: Content-Type,Authorization,If-None-Match,X-User-Agent
< access-control-allow-methods: GET,HEAD,OPTIONS
< access-control-allow-origin: *
< access-control-expose-headers: Content-Type,Warning,ETag,X-Pages,X-ESI-Error-Limit-Remain,X-ESI-Error-Limit-Reset
< access-control-max-age: 600
< allow: GET,HEAD,OPTIONS
< strict-transport-security: max-age=31536000
< x-esi-error-limit-remain: 99
< x-esi-error-limit-reset: 59
< x-esi-request-id: ac2e24a7-bd15-433e-aec5-5c804c66960b
```

### Body

Please provide the response body, feel free to scrub any opsec details you wish. E.g.:

```json
    {
    "error": "Unhandled internal error encountered!"
    }
```

## Expected

`200`

Along with a body containing the json of the character's information.

# Checklist

Check all boxes that apply to this issue:

- [X] Bug description is provided
- [X] Request path is provided
- [X] Response status code is provided
- [X] Response headers are provided
- [X] Response body is provided
- [X] Expected response is provided

**Comment by cvweiss** [comment]
Same issue as above:

    https://esi.evetech.net/v4/characters/504629349/

**Comment by StephenSwat** [comment]
I doubt this is an ESI issue. If you try to open these characters in game (by creating a custom `showinfo` URL, such as `<url=showinfo:1379//2029709492>Mystery Toon</url>`) the info window will hang and no character will be loaded. That's the same behaviour that you get when you try to show info for a completely random character ID. Perhaps these are characters that existed before and, instead of being moved to Doomheim, were deleted completely for whatever reason?

**Comment by cvweiss** [comment]
The fact in game is affected will help with debugging. I don't think these characters have been fully deleted, they have however had their names changed to the default <Race> Characer <ID>.

**Comment by Ionaru** [comment]
Interestingly https://imageserver.eveonline.com/Character/2029709492_256.jpg does have an image for this (deleted?) character.

**Comment by cvweiss** [comment]
As in completely gone, not just doomheimed? 

Reason I ask is this hasn't happened to characters in the past, and these characters have killmail records, so it will be difficult to pull their name(s), even if its just <Racial> Character <ID> and/or corporation history.

**Comment by ccp-zoetrope** [comment]
@cvweiss yes, it is odd and I have not really seen it before so I will look a bit more into why that is.

**Comment by CarbonAlabel** [comment]
@ccp-zoetrope see https://github.com/esi/esi-issues/issues/936 for how you handled the same thing about 6 months ago.

**Comment by ccp-zoetrope** [comment]
@CarbonAlabel always nice to have people around that have a way better memory than me 😆

**Comment by cvweiss** [comment]
@ccp-zoetrope while you're re-looking into this, can you reopen the ticket please?

**Comment by ccp-zoetrope** [comment]
Reopening until I can find out why characters were flat out deleted.

**Comment by ccp-zoetrope** [comment]
@cvweiss seems this was a bug with the first round of gdpr deletions and these characters will be restored. Please throw in any character IDs you know of that throw 404's.

**Comment by tsuthers-ccpgames** [comment]
Does this ticket still need to be open?

**Comment by cvweiss** [comment]
I'd say yes since the characters listed above were once valid entities and are now no longer valid entities. Previous discussions were that the characters were flat out deleted, rather than having the character's name changed.

**Comment by tsuthers-ccpgames** [comment]
Rgr, talking to people about it now

**Comment by ccp-pinky** [comment]
All the URLs in this ticket now work fine; I assumed that was already since your last comment @cvweiss , but clearly it wasn't. 

Can you confirm this has long since been fixed, or otherwise point out what characterIDs are missing?
